.github/workflows/win-cpu-arm64-build.yml

```diff
@@ -52,6 +52,13 @@ jobs:
         move ${{ env.ORT_PACKAGE_NAME }}/build/native/include ort/
         move ${{ env.ORT_PACKAGE_NAME }}/runtimes/win-arm64/native/* ort/lib/
 
+    - name: Install Rust
+      run: |
+        $exePath = "$env:TEMP\rustup-init.exe"
```

Comment:
There is no GHA action for this?

Reply:
Seems no official action for this, and some 3rd party actions only support linux and macos.